terraform/variables.tf

```diff
@@ -12,24 +12,30 @@ variable "remote_state_bucket" {
 }
 
 variable "state_prefix" {}
+variable "workspace_key_prefix" {}
+variable "state_file_name" {}
 
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
-variable "vpc_cidr" {
-  default = {
-    eu-west-2 = "10.75.0.0/16" #London
-  }
+# Required for access from existing Concourse instances
+variable "dev_management_cidrs" {
+  type    = list(string)
+  default = ["10.50.17.0/24","10.50.19.0/24","10.50.21.0/24"]
 }
-# These vpcs are configured for development. Preprod and prod are overridden in the vars file.
+
+# VPC ID read from terraform remote state.
 variable "vpc_id" {
-  default = {
-    eu-west-2 = "vpc-074ff55ed5182e144" #London
-  }
+  default     = ""
 }
 
 variable "environment" {}
-variable "internal_cidrs" {}
 variable "deploy_to" {}
 
+variable "docker_registry" {} # This value passed in from CI pipeline
+variable "release_version" {} # This value passed in from CI pipeline
+
+variable "docker_container_port" {
+  default = "10000"
```

Comment:
I presume this is a default application port? If so I'd remove the default as it'll cause issues

Reply:
This is actually the place where this value is defined - then used is various other places (ECS config, load balancer, etc.)

Reply:
If this is essentially defining the application port for the test data generator application then I don't see the point in defining it as a variable here as it's not going to change. I'd just define it in the main.tf locals section along with the name of the stack which is also hardcoded there

Reply:
Done as per suggestion - moved the variable to locals and also renamed it to `application_port` as per our other convo.